multiversx_sdk_cli/cli_shared.py

```diff
@@ -172,6 +175,28 @@ def prepare_nonce_in_args(args: Any):
         args.nonce = account.nonce
 
 
+def prepare_chain_id_in_args(args: Any):
+    if not args.chain and not args.proxy:
+        raise ArgumentsNotProvidedError("chain or proxy should be provided")
+
+    if args.chain and args.proxy:
+        proxy = ProxyNetworkProvider(args.proxy)
+        chain_id = proxy.get_network_config().chain_id
+
+        if args.chain != chain_id:
+            show_deprecation_warning("The chainID you have provided does not match the chainID you got from the proxy. Will use the value from the proxy.")
```

Comment:
`chain ID`.

Here, it should somehow be `show_warning` (it is not a deprecation warning per se).

Reply:
renamed the method to `show_warning`